app/src/main/java/org/fossasia/phimpme/gallery/adapters/MediaAdapter.java

```diff
@@ -50,6 +50,11 @@ public void updatePlaceholder(Context context) {
         placeholder = (BitmapDrawable) drawable;
     }
 
+    public ArrayList<Media> getList()
+    {
```

Comment:
@angmas1 move the opening bracket of the method next to the method declaration as it is in all other methods.